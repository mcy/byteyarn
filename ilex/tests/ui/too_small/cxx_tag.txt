R'c(foo)c' R'(foo)' Q'foo(bar)foo' Q'oo(bar)oo'
