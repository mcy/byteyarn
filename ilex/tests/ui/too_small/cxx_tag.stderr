error: expected at least 1 character in identifier, but found none
 --> too_small/cxx_tag.txt:1:14
  |
1 | R'c(foo)c' R'(foo)' Q'foo(bar)foo' Q'oo(bar)oo'
  |              ^ expected at least 1 here
  |
  = help: this appears to be an empty identifier
  = note: reported at: ilex/src/rt/emit2.rs:223:14

error: expected at least 3 characters in identifier, but found only 2
 --> too_small/cxx_tag.txt:1:38
  |
1 | R'c(foo)c' R'(foo)' Q'foo(bar)foo' Q'oo(bar)oo'
  |                                      ^^ expected at least 3 here
  |
  = note: reported at: ilex/src/rt/emit2.rs:223:14

error: aborting due to 2 errors
